bitnami/kube-prometheus/Chart.yaml

```diff
@@ -32,4 +32,4 @@ maintainers:
 name: kube-prometheus
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/kube-prometheus
-version: 8.10.1
+version: 8.10.2
```

Comment:
@su-tiko Can you please update your branch with the changes from the main branch? The current branch of contour is 8.10.2 not 8.10.1. So you should also update the version from 8.10.2 to 8.10.3.

Reply:
done!